ModularBungalow/clothing/smc.dm

```diff
@@ -21,6 +21,11 @@
 	name = "triage technician's jumpsuit"
 	icon_state = "triagetech"
 
+/obj/item/clothing/under/smc/triageofficer
+	desc = "A medic's sundicate jumpsuit with a hint of green. Helps stand out in the heat of battle."
```

Comment:
woah new 'sundicate' faction confirmed?????? you can't leak our secret ideas in the second line of the pr kirie!!!!!!!1!111111111!